conditional/templates/intro_eval_slideshow.html

```diff
@@ -12,6 +12,7 @@
     {% for m in members %}
     <section id="slide-{{m['uid']}}">
       <section>
+        <img class="eval-user-img" alt="{{m['uid']}}" src="https://profiles.csh.rit.edu/image/{{m['uid']}}" width="100" height="100" />
```

Comment:
Any reason you're doing the styles inline here (and also in `spring_eval_slideshow.html`) if you're already changing some CSS?

Reply:
Oops. Left that in by accident

Reply:
What do you mean? I don't see any inline css here? Only the width and height setting

Reply:
Right, that's what I'm referencing. I guess my wording was off, but it's still "styling" them. Just wondering why you didn't just set
```
    width: 100px;
    height: 100px;
```
instead. Guess this is more of a nitpick, though, that's all I found to comment on.